include/bluetooth/audio/audio.h

```diff
@@ -1516,6 +1516,45 @@ int bt_audio_unicast_group_create(struct bt_audio_stream *streams,
 				  uint8_t num_stream,
 				  struct bt_audio_unicast_group **unicast_group);
 
+/** @brief Add streams to a unicast group as a unicast client
+ *
+ *  This function can be used to add additional streams to a
+ *  bt_audio_unicast_group.
+ *
+ *  This can be called at any time before any of the streams in the
+ *  group has been started (see bt_audio_stream_ops.started()).
+ *  Once all streams in a group has been stopped
```

Comment:
Suggest to rephrase to "Additional streams can also be added using this function once all streams in the group has been stopped [...]" 
or "Additional streams can also be added after all streams in the groups have been stopped [...]"

Similar for the remove function.

Reply:
`have been stopped` implying that the callback needs to have been called before you can call this, which isn't true. I'm up for rephrasing it, but it should be more accurate than this suggestion.

Reply:
Now I see that @Casper-Bonde-Bose also had comments on this (I was reviewing commit by commit).

I fail to see the difference regarding callbacks between what I commented on and my proposed text.
The "has been stopped" is there in both.  (It should be changed to a "have", though - "streams" is plural.)

My intention was only to reorder the sentence, so that the addition of streams comes first, and the condition next, for consistency with the previous sentence.
- streams may be added before any streams in the group have been started
- streams may be also added after all streams in the group have been stopped

Reply:
> I fail to see the difference regarding callbacks between what I commented on and my proposed text.

> "Additional streams can also be added after all streams in the groups have been stopped [...]"

Implies that the streams have been stopped before this can be called, but it can also be called before they are started. Streams do not init to the the "stopped" state. 

But I think I see what you mean now. Let me try to rephrase it.

Reply:
Let me know if the current wording is better.